Dockerfile

```diff
@@ -70,9 +70,10 @@ RUN if ! getent group "$USER_GID"; then groupadd --gid "$USER_GID" dependabot ;
 ### RUBY
 
 # When bumping Ruby minor, need to also add the previous version to `bundler/helpers/v{1,2}/monkey_patches/definition_ruby_version_patch.rb`
-ARG RUBY_VERSION=3.1.2
+ARG RUBY_VERSION=3.1.3
 ARG RUBY_INSTALL_VERSION=0.8.5
 # Generally simplest to pin RUBYGEMS_SYSTEM_VERSION to the version that default ships with RUBY_VERSION.
+# TODO check CI to see what's being installed here before beign overwritten
```

Comment:
also bump it here to match:
https://github.com/dependabot/dependabot-core/blob/dd2c5dba914782e3acbd2b9f052af1b973347c88/common/dependabot-common.gemspec#L21

Reply:
Actually, looks like `ruby` `3.1.3` still bundles `rubygems 3.3.7`, at least per the output from CI:
```
#7 381.6 >>> Successfully installed ruby 3.1.3 into /usr/local
#7 382.5 Updating rubygems-update
#7 382.5 Successfully installed rubygems-update-3.3.7
#7 382.5 Installing RubyGems 3.3.7
#7 383.0   Successfully built RubyGem
#7 383.0   Name: bundler
#7 383.0   Version: 2.3.7
#7 383.0   File: bundler-2.3.7.gem
#7 383.0 Bundler 2.3.7 installed
#7 383.0 RubyGems 3.3.7 installed
#7 383.0 Regenerating binstubs
#7 383.0 Regenerating plugins
#7 383.0 
#7 383.0 
#7 383.0 
#7 383.0 ------------------------------------------------------------------------------
#7 383.0 
#7 383.0 RubyGems installed the following executables:
#7 383.0 	/usr/local/bin/gem
#7 383.0 	/usr/local/bin/bundle
#7 383.0 	/usr/local/bin/bundler
#7 383.0 
#7 383.1 RubyGems system software updated
```

Reply:
I think that's because we install that specific version in the `Dockerfile`. I think we probably want to remove that and let the version that comes with the Ruby we're using be used.